src/Core/Entities/User.cs

```diff
@@ -84,7 +84,7 @@ public string BillingName()
         return Name;
     }
 
-    public string OrganizationName()
+    public string GetName()
     {
         return Name;
```

Comment:
A user's `Name` is optional and could be `null`. If we're going to start displaying it on invoices, you might have to use their email as a fallback.